src/dom.ts

```diff
@@ -270,44 +270,31 @@ export function compareElementArea(a: HTMLElement, b: HTMLElement): number {
     return aArea - bArea
 }
 
-/** Computes a unique CSS selector for the current HTMLElement.
- *  Currently buggy, returns a unique selector but doesn't always correspond
- *  to the selector you should use for functions such as querySelectorAll().
- */
-export function getSelector(elem: HTMLElement) {
-   if (elem === null || elem === undefined)
-      return undefined
-
-   if (elem == document.body || !elem.parentNode)
-      return "body"
-
-   if (elem.id)
-      return "#" + elem.id
-
-   let parent = elem.parentNode as HTMLElement
-   let n = [].indexOf.call(parent.childNodes, elem) + 1
-   return getSelector(parent) + " :nth-child(" + n + ")"
-}
-
-export const hintworthy_js_elems = {}
+export const hintworthy_js_elems = []
 
 /** Adds or removes an element from the hintworthy_js_elems array of the
  *  current tab.
  *  This function is made available to the page's. We should be very careful
  *  about the input it accepts and what it does with it.
  */
-export function registerEvListenerAction(action: string, ev: string): void {
-   if (!(this instanceof HTMLElement))
+export function registerEvListenerAction(elem: EventTarget, add: boolean, event: string) {
+   if (!(elem instanceof Element))
       return
-
-   switch (ev) {
+   switch (event) {
       case "click":
-      case "hover":
+      case "mousedown":
+      case "mouseup":
       case "mouseover":
-         if (action == "addEventListener")
-            hintworthy_js_elems[getSelector(this)] = this
-         else
-            delete hintworthy_js_elems[getSelector(this)]
+      case "mouseout":
+      case "mouseleave":
+         let index = hintworthy_js_elems.indexOf(elem)
```

Comment:
Just realized this is buggy since you could
```js
addEventListener("click");
addEventListener("hover");
removeEventListener("hover");
```
Will fix this later.